components/forms/change-password/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useRef } from 'react'
+
+import styles from './styles.css'
+
+import { Card } from 'design'
+import { withGlobalStore } from 'store'
+
+const { IDP_URL } = process.env
+
+const ChangePassword = ({ className, store, tag = 'section' }) => {
```

Comment:
I do not think makind it being section here is a good idea. You could pass `className` and `tag` as rest props . Also, there is no reason to deny the user pass other HTML attributes.

Reply:
There is no need to pass other HTML attributes. If there is we can simply add it.